templates/init-d.ejs

```diff
@@ -14,7 +14,11 @@
 
 dir="<%- workingDirectory %>"
 user="<%- uid %>"
-cmd="<% Object.keys(env).forEach(function(key) {%><%- key %>=<%- env[key] %> <% }); %><%- nodeBin %> <%- startScript %><% flatArgs.forEach(function(arg) { %> <%- arg %><% }); %>"
+<% Object.keys(env).forEach(function (key) { %>
+<%- key %>=<%- env[key] %>
```

Comment:
is there a reason for performing the variable declaration in this form:

```bash
foo = bar
export foo
```

Could it just be:

```bash
export foo = bar
```

I don't know `init.d` syntax too well.